express/code/blocks/banner/banner.js

```diff
@@ -15,6 +15,84 @@ export default async function decorate(block) {
   const isBannerStandoutVariant = block.classList.contains('standout');
   const isBannerCoolVariant = block.classList.contains('cool');
 
+  // Check for background image classes
+  const backgroundImageClasses = [
+    'light-bg',
+    'blue-green-pink-bg',
+    'blue-bg',
+    'blue-pink-orange-bg',
+    'green-blue-red-bg',
+    'blue-purple-gray-bg',
+    'yellow-pink-blue-bg',
+  ];
+
+  let hasBackgroundImage = false;
+  for (const className of backgroundImageClasses) {
+    if (block.classList.contains(className)) {
+      hasBackgroundImage = true;
+      break;
+    }
+  }
+
+  // Load background variants CSS if needed
+  if (hasBackgroundImage) {
+    // Completely disable ALL banner.css files to prevent CSS regression
+    const allBannerCSS = document.querySelectorAll('link[href*="banner.css"]');
+    allBannerCSS.forEach((link) => {
+      link.disabled = true;
+      link.style.display = 'none';
+    });
+
+    // Also disable any inline styles that might interfere
+    const bannerBlocks = document.querySelectorAll('.banner');
+    bannerBlocks.forEach((bannerBlock) => {
+      if (backgroundImageClasses.some((cls) => bannerBlock.classList.contains(cls))) {
+        // Remove any inline styles that might interfere
```

Comment:
This breaks AEM block scoping standards by manipulating ALL banner blocks globally instead of just the current block.